test/test_request_validator.rb

```diff
@@ -79,6 +79,35 @@ def self.update
   end
 end
 
+class UserRelationshipMotherForm
```

Comment:
Since the form is for relationships, I think I'd prefer to see `Relationships` used in the class names.